### PR TITLE
Docs: Create thresholds.md

### PR DESCRIPTION
**Pull request by oddlittlebird**
**What this PR does / why we need it**: Creates documentation for the Thresholds section of the new Panel Edit.

**Special notes for your reviewer**: Please add any detail you think is appropriate and make sure everything I have noted is correct. It is all based on either my direct testing of a beta image or legacy documentation.

### CODE REVIEW

File: docs/sources/panels/thresholds.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+You can define thresholds one of two ways:
+* **Absolute** thresholds are defined based on a number. For example, 80 on a scale of 1 to 150.
+* **Percentage** thresholds are defined relative to minimum and maximums. For example, 80 percent.
+
+You can apply thresholds to the following visualizations:
+* Stat
+* Gauge
+* Bar gauge
+* Table
```
Comment (dprokop) [inline docs/sources/panels/thresholds.md:21]:
```suggestion
* Table
* Graph
```

File: docs/sources/panels/thresholds.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+1. Navigate to the panel you want to add a threshold to.
+1. Click the **Field** tab.
+1. Click **Add threshold**. 
+1. Grafana adds a threshold with suggested numerical and color values.
+1. Accept the recommendations or edit the new threshold.
+   * **Edit color:** Click the color dot you wish to change and then select a new color.
+   * **Edit number:** Click the number you wish to change and then enter a new number.
+   * **Thresholds mode -** Click the mode to change it for all thresholds on this panel.
+1. Click **Save** to save the changes in the dashboard.
```
Comment (dprokop) [inline docs/sources/panels/thresholds.md:44]: What's up with this numbers?

File: docs/sources/panels/thresholds.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+* Base = green
+* Mode = Absolute
+
+The **Base** value represents minus infinity. It is generally the “good” color.
+
+## Add a threshold
+
+You can add as many thresholds to a panel as you want. Grafana automatically sorts thresholds from highest value to lowest.
+
+1. Navigate to the panel you want to add a threshold to.
```
Comment (dprokop) [inline docs/sources/panels/thresholds.md:36]: Shit, this flow is not valid for Graph panel, as it uses legacy thresholds editor:/ we need to document that too

File: docs/sources/panels/thresholds.md (hunk 2 above)
Comment (oddlittlebird) [reply docs/sources/panels/thresholds.md:45]: It's called lazy numbering. When the markdown is rendered, it automatically makes numbers. This way we don't have to renumber if we add or delete a step in the middle. http://google.github.io/styleguide/docguide/style.html#use-lazy-numbering-for-long-lists

File: docs/sources/panels/thresholds.md (hunk 3 above)
Comment (oddlittlebird) [reply docs/sources/panels/thresholds.md:37]: Why can't we update it to the new editor?

Comment (oddlittlebird) [reply docs/sources/panels/thresholds.md:39]: Can we just update the Graph panel to the new editor?

**Comment by oddlittlebird** [comment]
@dprokop - I added graph threshold instructions, but I could not figure out what the Y-Axis field did. Neither option seemed to make any change in the visualization.

File: docs/sources/panels/thresholds.md (hunk 3 above)
Comment (dprokop) [reply docs/sources/panels/thresholds.md:39]: nope, it's an angular panel, so we have to wait

**Comment by dprokop** [comment]
@oddlittlebird regarding Y-axis. For each series you can which y-axis should it use. Click on the color icon in the legend, and from the popup click y-axis and the use right y axis:
image


Then the threshold depending on it's config will use either left or right axis

File: docs/sources/panels/thresholds.md (hunk 2 above)
Comment (dprokop) [reply docs/sources/panels/thresholds.md:47]: <3

**Review by dprokop — APPROVED** [review]

**Pull request merged**